test/test_database.rb

```diff
@@ -203,6 +203,20 @@ def test_new
       db.close if db
     end
 
+    def test_open
+      db = SQLite3::Database.open(':memory:')
+      assert db
```

Comment:
Can you please assert on the type of this object?

Reply:
I also changed the check for other places with just `assert db` and added one missing test - to ensure that [an instance of Database is yielded](https://github.com/sparklemotion/sqlite3-ruby/compare/adee4c7d2903e46318a4b7190590fdcd4ea50f28..0ef9a94d8f50122f59e45f2d7a535f02eefce5a3#diff-1a78f662d21da725643065914897a8dbd41fbd5893c3a90353382f367285330aR228-R235)

Reply:
Thank you!